tensilelite/Tensile/TensilePass.py

```diff
@@ -67,7 +67,11 @@ def _replaceActBranchLabel(module, labels):
     for item in module.items():
         if isinstance(item, Module):
             if "InsertActFunctionCallAddrCalc" in item.name:
-                labelLeft = labels[1:]
+                labelFirst = labels[0]
```

Comment:
This is for **_replaceActBranchLabel() always replaces label without postfix.
However we should check the label we'd really like to replace with**. @KKyang